pennylane_sphinx_theme/navbar.py

```diff
@@ -65,6 +65,10 @@
             },
         ],
     },
+    {
+        "name": "Get involved",

```

Comment:
Should "involved" be capitalized here? 🤔 
```suggestion
        "name": "Get Involved",

```

Reply:
No, not according to the design. I will talk with Will monday morning to confirm that it is not a mistake on their end.

<img width="644" alt="Screenshot 2024-01-05 at 6 26 03 PM" src="https://github.com/PennyLaneAI/pennylane-sphinx-theme/assets/53958929/09b966df-3007-4a53-94f9-55457d017516">

reference image above:

Reply:
Sounds good! Once we have confirmation either way, I can approve.

Reply:
@Mandrenkov - Yep, sentence case it is.